src/AccordionPanel.jsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+import Icon from './Icon';
+
+export const PANEL_CLASS = 'accordionPanel';
+export const ACTIVEPANEL_CLASS = 'accordionPanel--active';
+
+/**
+ * @module AccordionPanel
+ */
+class AccordionPanel extends React.Component {
+	constructor(props){
+		super(props);
+
+		this.state = {
+			isOpen: this.props.isOpen
+		};
+
+		this._handleToggle = this._handleToggle.bind(this);
+	}
+
+	/**
+	 * @parm {Boolean} isOpen (is panel open)
+	 * @returns {Number} panel height
+	 */
+	getHeight(isOpen) {
+		return `${isOpen * this.contentEl.getBoundingClientRect().height}px`;
+	}
+
+	/**
+	 * @returns {undefined}
+	 *
+	 * Updates state to toggle `AccordionPanel` open and closed
+	 */
+	_handleToggle(){
+		const newState = !this.state.isOpen;
+		const { setClickedPanel } = this.props;
```

Comment:
nitpick, but since this var is only used once in an `if` test, I'd skip the destructuring and just write `if (this.props.setClickedPanel)`